index.html

```diff
@@ -63,7 +64,7 @@
         <strong>Connection error</strong> The client was unable to connect to the WeeChat relay
       </div>
       <div class="alert alert-danger" ng-show="sslError">
-        <strong>Secure connection error</strong> A secure connection with the WeeChat relay could not be initiated. This is most likely because your browser does not trust your relay's certificate. Please read the encryption instructions below!
+        <strong>Secure connection error</strong> A secure connection with the WeeChat relay could not be initiated. This is most likely because your <span ng-hide="utils.isCordova()">browser</span><span ng-hide="!utils.isCordova()">device</a> does not trust your relay's certificate. <span ng-hide="utils.isCordova()">Please read the encryption instructions below!</span><span ng-hide="!utils.isCordova()">It's easiest to use a trusted certificate, and remember to include all intermediate CAs. For more info, please read the encryption instructions below!</span>
```

Comment:
please use one-time bindings for the cordova checks (prefix with `::`) to prevent creation of watchers. Since this check cannot change at runtime it's safe and will make things faster